spanner/api/Spanner.Samples.Tests/UpdateUsingDmlReturningAsyncPostgreTest.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright 2023 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Cloud.Spanner.Data;
+using System.Threading.Tasks;
+using Xunit;
+
+[Collection(nameof(SpannerFixture))]
+public class UpdateUsingDmlReturningAsyncPostgreTest
+{
+    private readonly SpannerFixture _spannerFixture;
+
+    public UpdateUsingDmlReturningAsyncPostgreTest(SpannerFixture spannerFixture)
+    {
+        _spannerFixture = spannerFixture;
+    }
+
+    [Fact]
+    public async Task TestUpdateUsingDmlReturningAsyncPostgre()
+    {
+        await PrepareDataAsync();
+        var sample = new UpdateUsingDmlReturningAsyncPostgreSample();
+        var updatedMarketingBudget = await sample.UpdateUsingDmlReturningAsyncPostgre(_spannerFixture.ProjectId, _spannerFixture.InstanceId, _spannerFixture.PostgreSqlDatabaseId);
+
+        Assert.Single(updatedMarketingBudget);
+        Assert.Equal(20000, updatedMarketingBudget[0]);
+    }
+    private async Task PrepareDataAsync()
+    {
+        string alterStatement = "ALTER TABLE Albums ADD COLUMN MarketingBudget BIGINT";
```

Comment:
Instead of ALTER statement here, can we not add this column where the table is being created?
And we can just insert the data here. Please note we can use BatchCommand rather than inserting multiple times.